server/chat-plugins/info.js

```diff
@@ -2519,6 +2526,44 @@ const pages = {
 		}
 		return buf;
 	},
+	globalpunishments(query, user) {
+		this.title = 'Global Punishments';
+		let buf = "";
+		if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
+		if (!this.can('lock')) return;
+		const sortedPunishments = Punishments.getGlobalPunishments().sort((a, b) => {
+			// Ascending order
+			return a.expiresIn - b.expiresIn;
+		});
+		if (sortedPunishments.length) {
+			buf += `<div class="pad"><h2>List of active global punishments:</h2>`;
+			buf += `<table style="border: 1px solid black; border-collapse:collapse; width:100%;">`;
```

Comment:
This is a lot of custom table styling. Was something wrong with `<table class="table">`?

Reply:
Huh I guess it's `.ladder table`

Reply:
I just used `<div class="ladder pad"`. I didn't know the existence of it, it's pretty neat!